initial_website_build/initial_website_build.info.yml

```diff
@@ -0,0 +1,29 @@
+name: 'Initial website build'
+description: 'This is the first build of the RWAHS site, it is not refined yet and still requires considerable cleaning up of theming and content, but all the basic config has been completed.'
```

Comment:
I'd replace this with `This feature contains the base configuration of the RWAHS site.`

The reset of this would be part of the description of the pull request, but the feature itself is something that evlolves over time.